AppStore/German/Description.iOS.txt

```diff
@@ -0,0 +1,29 @@
+Hast du dich jemals gefragt, "Wer ist dieser Schauspieler?", oder vielleicht "Wann wurde dieser Film veröffentlicht?", oder "Wie war der Titel dieser einen Episode meiner Lieblingsserie?" Vielleicht bist du ein Filmkenner und fragst dich, "Wie alt war diese Person, als dieser Film veröffentlicht wurde?"
+
+Wolltest du schon immer solche Fragen beantworten können, *ohne* alle fünf Sekunden irgendwo aufgefordert zu werden dich anzumelden? Ohne Werbung und ohne automatisch abgespielte Videos? Ohne auf ein multinationales Konglomerat angewiesen zu sein, das dir Dinge verkaufen möchte?
+
+Dann wirst du Callsheet lieben.
+
+Callsheet ist der beste Weg, um schnell Informationen über Besetzung und Crew in Fernsehserien und Filmen zu finden. Es geht mit deiner Zeit respektvoll um, versucht nicht, dir Dinge zu verkaufen, die du nicht brauchst, und ist darauf ausgelegt, deine Fragen schnell und unkompliziert zu beantworten.
+
+Callsheet bietet auch einige neuartige Funktionen, die deine bisherige Film-/Seriendatenbank-App nicht hat:
+
+• Spoiler vermeiden beim Anschauen von TV-Serien, indem optional Charakternamen, Episodenzahlen, Episodentitel oder Episodenvorschaubilder ausgeblendet werden.
+  Lass dich nicht durch Hinweise auf eine geheime Identität eines Charakters spoilern oder darauf, dass sie die Serie schnell verlassen, indem du diese Hinweise ausblendest.
+• Schneller Zugriff auf Wissenswertes, Wikipedia, Altersbeschränkungen und Informationen, wo man den Film oder die Serie anschauen kann (bereitgestellt von JustWatch).
+• Pinne TV-Sendungen, Filme oder Personen für schnellen Zugriff an.
+• Finde Dinge, nach denen du zuvor gesucht hast, mit den Funktionen "Letzte Suchen" und "Suchhistorie".
+• Leichter Zugriff auf das Alter der Cast-Mitglieder *bei der Veröffentlichung eines Films* direkt neben der Besetzungsliste
+• Unkomplizierte Ansicht des Alters einer Person neben jedem Eintrag in ihrer Filmografie
+• Direkte Anzeige der Körpergröße der Schauspielenden (wenn möglich)
+• Unterstützt den Dunkelmodus
+• Unterstützt VoiceOver
+• Alternative App-Symbole für Abonnenten
+• Automatische Integration mit der wunderbaren Channels-App - keine Anmeldung erforderlich!
+• Experimentelle Integration mit Plex, ebenfalls ohne Anmeldung.
+
+Callsheet wird von einer Person geschrieben, die WIRKLICH darauf bedacht ist, deine Zeit zu respektieren. Callsheet ist darauf ausgelegt, dass du schnell und unkompliziert das findest, was du brauchst, und dann wieder zu dem Film oder der Serie zurückkehren kannst, die dir wirklich am Herzen liegt. Keine Ablenkungen und keine Aufforderungen, dich irgendwo anzumelden.
```

Comment:
Maybe replace the second Callsheet with just "Es" oder "Die App"

```...bedacht ist, deine Zeit zu respektieren. Es ist darauf ausgelegt...```

```...bedacht ist, deine Zeit zu respektieren. Die App ist darauf ausgelegt...```

Reply:
```suggestion
Callsheet wird von einer Person geschrieben, die WIRKLICH darauf bedacht ist, deine Zeit zu respektieren. Callsheet ist darauf ausgelegt, dass du schnell und unkompliziert das findest, was du suchst, und dann wieder zu dem Film oder der Serie zurückkehren kannst, die dir wirklich am Herzen liegt. Keine Ablenkungen und keine Aufforderungen, dich irgendwo anzumelden.
```

Reply:
Good suggestions!